graylog2-web-interface/src/views/components/datatable/DataTableEntry.test.jsx

```diff
@@ -1,20 +1,20 @@
 // @flow strict
 import * as React from 'react';
-import * as Immutable from 'immutable';
+import { List, OrderedSet } from 'immutable';
```

Comment:
Everywhere else we use 'immutable' we use `import * as Immutable from 'immutable';`

Should we not destruct here to keep consistent with the rest of the codebase?

Reply:
We do not use one form consistently across the codebase (see `ViewState`, `ViewTransformer`, `CopyWidgetToDashboard` for examples of importing individual parts. Both ways have pros and cons (one being explicit, the other being shorted and allowing tree shaking), so I would not vote for enforcing one style.